integration_test/install_test.go

```diff
@@ -27,12 +28,68 @@ const (
 
 func TestKrewInstall(t *testing.T) {
 	skipShort(t)
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	if err := test.Krew("install", validPlugin); err == nil {
+		t.Fatal("expected to fail without initializing the index")
+	}
+
+	test = test.WithIndex()
+	if err := test.Krew("install"); err == nil {
+		t.Fatal("expected failure without any args or stdin")
+	}
+
+	test.Krew("install", validPlugin).RunOrFailOutput()
+	test.AssertExecutableInPATH("kubectl-" + validPlugin)
+}
+
+func TestKrewInstallReRun(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test.WithIndex().Krew("install", validPlugin).RunOrFail()
+	test.WithIndex().Krew("install", validPlugin).RunOrFail()
+	test.AssertExecutableInPATH("kubectl-" + validPlugin)
+}
+
+func TestKrewInstall_MultiplePositionalArgs(t *testing.T) {
+	skipShort(t)
 
 	test, cleanup := NewTest(t)
 	defer cleanup()
 
-	test.WithIndex().Krew("install", validPlugin).RunOrFailOutput()
+	test.WithIndex().Krew("install", validPlugin, validPlugin2).RunOrFailOutput()
```

Comment:
```suggestion
	test.Krew("install", validPlugin, validPlugin2).RunOrFailOutput()
```
If you want to keep the symmetry, this is fine, but it's not necessary.